CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ## [Unreleased]
 
 ### Added
+- rulebook and Drools squared accessor syntax (AAP-10738, DROOLS-7475)
```

Comment:
Minor: as this is user faced I think it should not contain internal references, just the description of the feature. This is how we are doing it until now.

Reply:
done with 081421a 👍